idea/src/org/jetbrains/kotlin/idea/intentions/branchedTransformations/IfThenUtils.kt

```diff
@@ -299,19 +299,23 @@ internal fun KtIfExpression.buildSelectTransformationData(): IfThenToSelectData?
 internal fun KtExpression?.isClauseTransformableToLetOnly() =
     this is KtCallExpression && resolveToCall()?.getImplicitReceiverValue() == null
 
-internal fun KtIfExpression.shouldBeTransformed(): Boolean {
+internal fun KtIfExpression.shouldBeTransformed(transformToSafeAccess: Boolean = false): Boolean {
```

Comment:
I think this parameter should be always true (yes, even for `IfThenToElvisInspection`), so it's not required.